src/peer-routing/types.d.ts

```diff
@@ -0,0 +1,10 @@
+export = PeerRouting;
+
+import PeerId from 'peer-id'
+import Multiaddr from 'multiaddr'
+
+declare class PeerRouting {
+  constructor (options?: Object);
+  findPeer (peerId: PeerId, options?: Object): Promise<{ id: PeerId, multiaddrs: Multiaddr[] }>;
```

Comment:
the `{id,multiaddrs}` appears twice here again, further suggests it should be it's own `Provider` type